packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../../globals/imports';
+
+@mixin contentblockcards-large-spacing {
+  padding-top: $carbon--layout-05;
+  padding-bottom: $carbon--layout-07;
+
+  .#{$prefix}--content-block__heading {
+    margin-bottom: $carbon--layout-05;
+  }
+}
+
+@mixin content-block-cards {
+  .#{$prefix}--content-block-cards {
+    .#{$prefix}--content-block {
+      padding-top: $carbon--layout-03;
+      padding-bottom: $carbon--layout-05;
+      @include carbon--breakpoint('md') {
+        padding-top: $carbon--layout-03;
+        padding-bottom: $carbon--layout-06;
+      }
+      @include carbon--breakpoint('lg') {
+        @include contentblockcards-large-spacing;
+      }
+      @include carbon--breakpoint('xlg') {
+        @include contentblockcards-large-spacing;
+      }
+      @include carbon--breakpoint('max') {
+        @include contentblockcards-large-spacing;
```

Comment:
hmm have a feeling we don't need to specify all these breakpoints

Reply:
I believe having this

```
@include carbon--breakpoint('lg') {
        @include contentblockcards-large-spacing;
}
```
should be good 

`carbon--breakpoint('lg')` is the same as `@media (min-width: (whatever rem lg breakpoint is))`

Reply:
@greeshmareddyg can you take a look?

Reply:
@annawen1 @jeffchew applying only lg breakpoint now..